wormhole-connect/src/views/v2/Bridge/SwapInputs/index.tsx

```diff
@@ -33,12 +33,13 @@ function SwapInputs() {
     fromChain &&
     !config.chains[fromChain]?.disabledAsDestination &&
     toChain &&
-    !config.chains[toChain]?.disabledAsDestination;
+    !config.chains[toChain]?.disabledAsSource;
```

Comment:
I think this was a copy/paste error, or am I missing anything here?

Reply:
came from here https://github.com/wormhole-foundation/wormhole-connect/commit/5c13b8d1aea40d1ee896deb4c0e7a375cffdbb02#diff-50062b7f121564dcafd21f3ee9350314aae20a3a4b2487eff668042c0d358587R36